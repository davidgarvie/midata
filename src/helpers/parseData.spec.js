const parseData = require("./parseData");
const data = require("./__mocks__/data");

describe("parseData", () => {
  test("should throw if not provided with any data", () => {
    expect(() => parseData()).toThrow();
  });

  test("should parse data", () => {
    expect(parseData(data)).toMatchSnapshot();
  });
});
