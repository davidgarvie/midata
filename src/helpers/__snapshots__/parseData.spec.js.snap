// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`parseData should parse data 1`] = `
Array [
  Object {
    "Balance": "+£90.00",
    "Date": "08/09/2018",
    "Debit/Credit": "-£10.00",
    "Merchant/Description": "SOME MOCK DATA",
    "Type": "PAYMENTS",
  },
  Object {
    "Balance": "+100.00",
    "Date": "07/09/2018",
    "Debit/Credit": "-£5.00",
    "Merchant/Description": "CARD PAYMENT TO ACME LTD",
    "Type": "CARD PAYMENT",
  },
]
`;
